dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
+WORKDIR /knights
+COPY package.json ./
+RUN npm install
+COPY . .
+EXPOSE 4000
```

Comment:
The port specified here is 4000 yet the app is now running on port 5000 as a default. Please change the default app port to 4000.

Reply:
I have changed it